app/assets/stylesheets/govuk_elements_extensions.scss

```diff
@@ -176,13 +173,14 @@ section {
   border: 1px solid $grey-2;
 }
 
+input[disabled],
 input[disabled="disabled"],
 input[readonly="readonly"],
 textarea[disabled],
 textarea[disabled="disabled"],
 textarea[readonly="readonly"] {
   opacity: 1;
-  background-color: $grey-4;
+  background-color: #eee;
```

Comment:
I would maybe abstract this to a colour variable. Just so we have a list of all colours in one place.